cs-amazon/src/main/java/io/cloudslang/content/amazon/factory/InputsWrapperBuilder.java

```diff
@@ -69,10 +77,13 @@ private InputsWrapperBuilder() {
         return wrapper;
     }
 
-    private static HttpClientInputs getHttpClientInputs(CommonInputs commonInputs) {
+    @SafeVarargs
+    private static <T> HttpClientInputs getHttpClientInputs(CommonInputs commonInputs, T... builders) {
         HttpClientInputs httpClientInputs = new HttpClientInputs();
 
-        httpClientInputs.setUrl(InputsUtil.getUrlFromApiService(commonInputs.getEndpoint(), commonInputs.getApiService()));
+        String prefix = getPrefix(builders);
+
+        httpClientInputs.setUrl(InputsUtil.getUrlFromApiService(commonInputs.getEndpoint(), commonInputs.getApiService(), prefix));
```

Comment:
Why isn't this just httpClientInputs.setUrl(commonInputs.getEndpoint()); since the endpoint from the commonInputs should be valid and not empty (since it is supposed to be build with the method withEndpoint)